### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ParameterTest/DateTimeVariantTest.cs [hunk 1]
```diff
@@ -102,7 +102,7 @@ private static void TestSimpleParameter_Type(object paramValue, string expectedT
             catch (Exception e)
             {
                 if (IsExpectedException(e, paramValue, expectedTypeName, expectedBaseTypeName))
-                    LogMessage(tag, "[EXPECTED EXPECTION] " + e.Message);
+                    LogMessage(tag, "[EXPECTED EXCEPTION] " + e.Message);
```
Comment (cheenamalhotra) [inline src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ParameterTest/DateTimeVariantTest.cs:105]: Please also update this is [BSL files](https://github.com/dotnet/SqlClient/search?q=EXPECTION&unscoped_q=EXPECTION)

File: src/Microsoft.Data.SqlClient/tests/FunctionalTests/SqlConnectionBasicTests.cs [hunk 2]
```diff
@@ -77,7 +77,7 @@ public void SqlConnectionInvalidParameters()
             Assert.Throws<ArgumentException>(() => new SqlConnection("Timeout= null;"));
             Assert.Throws<ArgumentException>(() => new SqlConnection("Timeout=1 1;"));
             Assert.Throws<ArgumentException>(() => new SqlConnection("Timeout=1a;"));
-            Assert.Throws<ArgumentException>(() => new SqlConnection("Integrated Security=truee"));
+            Assert.Throws<ArgumentException>(() => new SqlConnection("Integrated Security=true"));
```
Comment (cheenamalhotra) [inline src/Microsoft.Data.SqlClient/tests/FunctionalTests/SqlConnectionBasicTests.cs:80]: I think this was intentionally misspelled.. now it doesn't throw exception 😏

Comment (SimonCropp) [reply src/Microsoft.Data.SqlClient/tests/FunctionalTests/SqlConnectionBasicTests.cs]: ahh yes. silly mistake on my part. fixed

File: src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ParameterTest/DateTimeVariantTest.cs (hunk 1 above)
Comment (SimonCropp) [reply src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ParameterTest/DateTimeVariantTest.cs:105]: should be done now

**Comment by SimonCropp** [comment]
re-based on master

**Comment by SimonCropp** [comment]
do u want more of this kind of PRs?

**Comment by cheenamalhotra** [comment]
@SimonCropp 

Sure! Code improvements are always welcome :)

**Pull request merged**